build.cake

```diff
@@ -153,11 +167,9 @@ Task("nuget")
 		.SetConfiguration(CONFIGURATION)
 		.SetVerbosity(VERBOSITY)
 		.SetMaxCpuCount(0)
-		.WithRestore()
+		.WithProperty("NoBuild", "true")

```

Comment:
Don't build on the Pack - speeds things up as it doesn't have to check for Restore and Build.

Reply:
Cool trick